static/js/settings_panel_menu.js

```diff
@@ -19,6 +19,7 @@ class SettingsPanelMenu {
 
         this.main_elem.on("click", "li[data-section]", (e) => {
             const section = $(e.currentTarget).attr("data-section");
+            popovers.hide_all();
```

Comment:
This sort of change doesn't make sense to put in a click handler -- since then if we add some other UI widget that can change the view, it won't run.  Does this belong in `activate_section_or_default` instead?